code/modules/mob/living/carbon/carbon.dm

```diff
@@ -31,6 +31,10 @@
 /mob/living/carbon/Move(NewLoc, direct)
 	. = ..()
 	if(.)
+		if(m_intent == MOVE_INTENT_RUN && prob(1))
+			KnockDown(2)
```

Comment:
0.2 seconds.

Not important due to the nature of the pr but it's good for future reference.